ga4gh/backends.py

```diff
@@ -365,18 +365,23 @@ class Backend(object):
     """
     Superclass of GA4GH protocol backends.
     """
-    def __init__(self, dataDir, Dataset):
+
+    def __init__(self, dataDir, VariantSet):
```

Comment:
the capital V. This is a variable name, not a class name, and as such should not be capitalized based on the conventions of the rest of the code.

At least, that was my nitpick.